sormas-backend/src/main/java/de/symeda/sormas/backend/sormastosormas/entities/immunization/ReceivedImmunizationProcessor.java

```diff
@@ -43,19 +43,23 @@
 	public ReceivedImmunizationProcessor() {
 	}
 
+	private ImmunizationFacadeEjb.ImmunizationFacadeEjbLocal immunizationFacadeEjb;
+
 	@Inject
 	protected ReceivedImmunizationProcessor(
+		ImmunizationFacadeEjb.ImmunizationFacadeEjbLocal facadeEjb,
 		ImmunizationService service,
 		UserService userService,
 		ConfigFacadeEjb.ConfigFacadeEjbLocal configFacade,
 		SormasToSormasImmunizationDtoValidator validator) {
 		super(service, userService, configFacade, validator);
+		this.immunizationFacadeEjb = facadeEjb;
 	}
 
 	@Override
 	public void handleReceivedData(SormasToSormasImmunizationDto sharedData, Immunization existingData) {
 		updateReportingUser(sharedData.getEntity(), existingData);
-		handleIgnoredProperties(sharedData.getEntity(), ImmunizationFacadeEjb.toDto(existingData));
+		handleIgnoredProperties(sharedData.getEntity(), immunizationFacadeEjb.toDto(existingData));
```

Comment:
Not certain, what is the reason that here constructor injection is used to get the local Ejb while in the other processor it is used as Ejb, see `ReceivedEventProcessor` for example.